velox/exec/SortBuffer.cpp

```diff
@@ -145,23 +143,21 @@ void SortBuffer::noMoreInput() {
 
     VELOX_CHECK_NULL(spillMerger_);
     spillMerger_ = spiller_->startMerge();
-    spillSources_.resize(outputBatchSize_);
```

Comment:
Where we reset these two buffers? thanks!

Reply:
This is resize not reset

As for the reset of the those two buffers, from my understanding, we don't, we just override in the next iteration: https://github.com/facebookincubator/velox/blob/c80d9b4e914921f2368551f13f50526fff48b971/velox/exec/SortBuffer.cpp#L312

Reply:
Moved to `SortBuffer::getOutput`